src/main/resources/avro/bdg.avdl

```diff
@@ -64,6 +64,28 @@ record Contig {
   union { null, int } referenceIndex = null;
 }
 
+record RecordGroupMetadata {
+  /**
+   Record group identifier.
+   */
+  union { null, string } recordGroupName = null;
```

Comment:
The prefix in all the fields seems redundant.  And this record could just be called RecordGroup, no?